python_cmake_module/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+cmake_minimum_required(VERSION 2.8.3)
+
+project(python_cmake_module)
+
+find_package(ament_cmake REQUIRED)
+
+list(INSERT CMAKE_MODULE_PATH 0 "${CMAKE_CURRENT_SOURCE_DIR}/cmake/Modules")
+find_package(PythonExtra MODULE)
```

Comment:
But this CMake file seem to not use the result anywhere to why find it in the first place?